tests/acceptance/TestHelpers/OcisConfigHelper.php

```diff
@@ -120,4 +120,20 @@ public static function startOcis(): ResponseInterface {
 		$url = self::getWrapperUrl() . "/start";
 		return self::sendRequest($url, "POST");
 	}
+
+	/**
+	 * this method stops the running oCIS instance,
+	 * restarts oCIS without specific services,
+	 * and then starts the excluded services separately.
+	 *
+	 * @param string $service
+	 * @param array $envs
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function startService(string $service, array $envs): ResponseInterface {
```

Comment:
env can have default value
```suggestion
	public static function startService(string $service, array $envs = []): ResponseInterface {
```